src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -1849,7 +1848,7 @@ GetRoleSpecObjectForGrantStmt(Oid roleOid)
 /*
  * SetLocalEnableDependencyCreation sets the enable_object_propagation locally
  */
-static void
+void
 SetLocalEnableDependencyCreation(bool state)
 {
 	set_config_option("citus.enable_object_propagation", state == true ? "on" : "off",
```

Comment:
I was a bit confused by the name of the function vs. the name of the GUC.

What happens to this function post-#5678?

Reply:
It becomes `static void SetLocalEnableMetadataSync(bool state);`  here :https://github.com/citusdata/citus/pull/5678/files#diff-c30b7a7cd099f75c061f2f9ff042366ccd8467ad7eb47c1a39c9fcc9d6ac30abR108